### PR TITLE
Origin/feat internal ids for account / teams / users

### PR DESCRIPTION
**Pull request by fogelito**
## What does this PR do?
- We are adding Internal ids for subqueries table relations (sessions, tokens, membership).
- Changing Indexes to use new attributes.
- Change PDOPool settings for the connection (PDO::ATTR_DEFAULT_FETCH_MODE => PDO::FETCH_ASSOC) globally

## Test Plan
Change some tests according to createdAt, updatedAt new attributes.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
         "utopia-php/cache": "0.6.*",
         "utopia-php/cli": "0.12.*",
         "utopia-php/config": "0.2.*",
-        "utopia-php/database": "0.17.*",
+        "utopia-php/database": "dev-bug-last-internal-id as 0.18.1",
```
Comment (TorstenDittmann) [inline composer.json:48]: Just a reminder for us to use a tag before merging 👍🏻

File: docker-compose.yml [hunk 2]
```diff
@@ -236,6 +236,7 @@ services:
     volumes:
       - ./app:/usr/src/code/app
       - ./src:/usr/src/code/src
+      - ./vendor/utopia-php/database:/usr/src/code/vendor/utopia-php/database
```
Comment (TorstenDittmann) [inline docker-compose.yml:239]: This shouldn't be merged 👍🏻

File: tests/e2e/Services/Database/DatabaseCustomServerTest.php [hunk 3]
```diff
@@ -792,7 +792,7 @@ public function testIndexLimitException()
         // testing for indexLimit = 64
         // MariaDB, MySQL, and MongoDB create 3 indexes per new collection
         // Add up to the limit, then check if the next index throws IndexLimitException
-        for ($i = 0; $i < 61; $i++) {
+        for ($i = 0; $i < 59; $i++) {
```
Comment (TorstenDittmann) [inline tests/e2e/Services/Database/DatabaseCustomServerTest.php:795]: Lets update the comment above regarding the created indexes.

Comment (fogelito) [reply tests/e2e/Services/Database/DatabaseCustomServerTest.php]: Sure!

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**